test/config/customMatchers/toEqualGraphQLErrorType.ts

```diff
@@ -0,0 +1,9 @@
+import { GraphQLFormattedError } from "graphql";
+
+export const toEqualGraphQLErrorType = (
```

Comment:
lo cambiaria por "to include graphql error type", y también pasaría a usar un .includes en vez de un [0] hardcodeado